README.md

```diff
@@ -1,52 +1,188 @@
-## BEGIN - TO DELETE TILL END
+# reshuffle-github-connector
 
-THIS IS A TEMPLATE REPO FOR NEW RESHUFFLE CONNECTORS
-1. Create a new connector repo from this template using this link https://github.com/reshufflehq/reshuffle-template-connector/generate
-2. Clone the repo locally
-3. Rename all occurrences of _CONNECTOR_NAME_
-4. `npm install`
-5. `npm run build:watch`
-6. Implement your events/actions in `src/index.ts`
-7. `npm run lint`
-8. Push your code
-9. Go to https://app.circleci.com/projects/project-dashboard/github/reshufflehq/
-    a. You should see your new connector repo
-    b. click on `Set Up Project` for the repo
-    c. click on `Use Existing Config`
-    d. click on `Start Building`
+[Code](https://github.com/reshufflehq/reshuffle-github-connector) | [npm](https://www.npmjs.com/package/reshuffle-github-connector) | [Code sample](https://github.com/reshufflehq/reshuffle/tree/master/examples/github)
 
-10. If circle CI checks are all green, you are all set!
+`npm install reshuffle-github-connector`
 
-// Keep documentation template below
+This connector uses the [Octokit GitHub REST API](https://github.com/octokit/rest.js) package.
 
-## END
+### Reshuffle GitHub Connector
 
-# reshuffle-_CONNECTOR_NAME_-connector
+This connector provides a connector for [GitHub](https://www.github.com).
 
-### Reshuffle _CONNECTOR_NAME_ Connector
+The following example listens to all GitHub events on a repository:
 
-This connector provides <description>.
+```js
+const { Reshuffle } = require('reshuffle')
+const { GitHubConnector } = require('reshuffle-github-connector')
+
+const app = new Reshuffle()
+const connector = new GitHubConnector(app, {
+  token: process.env.TOKEN,
+  owner: process.env.OWNER,
+  baseUrl: process.env.BASE_URL,
+  repo: process.env.REPO,
+})
+
+connector.on({ githubEvent: '*' }, (event) => {
+  console.log('GitHub Event: ', event)
+})
+
+app.start()
+```
+
```

Comment:
We need to add a table of contents in the readme. I got that one twice in shopify and asana today :)